enumerables.rb

```diff
@@ -238,6 +238,8 @@ def multiply_els(array)
 # puts "inject f : #{e.my_inject { |sum, number| sum * number }}\n\n"
 # puts "inject g : #{d.inject}\n\n"
 # puts "inject g : #{d.my_inject(11)}\n\n"
+# ar = [1, 2, 3, 4]
+# puts "{#{ar.my_any?{|v| v > 100}}"
```

Comment:
Make sure to remove the commented code from you file. They make you code look untidy